lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -198,11 +198,10 @@ def call(params)
                                            do_not_touch_users_that_are_not_mentioned: do_not_touch_users_that_are_not_mentioned,
                                            create_non_existing_user_groups: create_non_existing_user_groups)
                     when 'sync_multiple_projects_based_on_pid'
-                      new_users.group_by { |u| u[:pid] }.flat_map do |project_id, users|
+                      new_users.group_by { |u| u[:pid] }.flat_map.pmap do |project_id, users|
                         begin
-                          project = client.projects(project_id)
-                          fail "You (user executing the script - #{client.user.login}) is not admin in project \"#{project_id}\"." unless project.am_i_admin?
-                          project.import_users(users,
+                          current_project = client.projects(project_id)
+                          current_project.import_users(users,
```

Comment:
Similar code found in synchronize_users.rb:193, synchronize_users.rb:204, synchronize_users.rb:221, synchronize_users.rb:247, synchronize_users.rb:265, synchronize_users.rb:292, synchronize_users.rb:334 (mass = 112)